HW_2/task_4.py

```diff
@@ -0,0 +1,33 @@
+"""
+Задайте список из N элементов, заполненный целыми числами из промежутка [-N, N].
+Найдите произведение элементов на индексах, хранящихся в файле indexes.txt (в одной строке один индекс).
+Решение должно работать при любом натуральном N.
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+"""
+
+from random import randint
+
+def get_numbers(n):
+    return [randint(-n/2, n) for i in range(-n, n + 1)]
```

Comment:
Почему `randint(-n/2, n)`? По условиям нужно убрать числа из промежутка [-N, N], а размерность должна быть `N`:
```suggestion
    return [randint(-n, n) for i in range(n)]
```